.eslintrc

```diff
@@ -1,3 +1,25 @@
 {
-  "extends": "smaato"
+  "extends": "smaato",
+  "env": {
+    "browser": true,
+  },
+  "rules": {
+    "class-methods-use-this": 0,
+    "consistent-return": 0,
+    "comma-dangle": [2, {
+      "arrays": "always-multiline",
+      "objects": "always-multiline",
+      "imports": "always-multiline",
+      "exports": "always-multiline",
+      "functions": "ignore",
+    }],
+    "import/extensions": 0,
+    "import/no-extraneous-dependencies": 0,
+    "jsx-a11y/no-static-element-interactions": 0,
+    "react/forbid-prop-types": 0,
+    "react/jsx-filename-extension": 0,
+    "react/no-array-index-key": 0,
```

Comment:
So as discussed, I had to find a reasonable way of either fixing or adding the rule to the exclusions. The smaato own configs aren't really changing any rules, instead we can clean up over time, so all those exclusions will be fixed in future anyway.